.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -85,6 +85,11 @@ jobs:
         sudo rsync -av ../extract/usr/include/ /usr/aarch64-linux-gnu/include/
         sudo rsync -av ../extract/lib/aarch64-linux-gnu/ /usr/aarch64-linux-gnu/lib/
 
+    - name: Setup cmake
+      uses: jwlawson/actions-setup-cmake@v2
+      with:
+        cmake-version: '3.26.4'
```

Comment:
This doesn't seem to uninstall the old one or something:
```
CMake Error: Could not find CMAKE_ROOT !!!
CMake has most likely not been installed correctly.
Modules directory not found in
/opt/actions-runner/_work/_tool/cmake/3.26.4/arm64/cmake-3.26.4-linux-x86_64/share/cmake-3.16
```

Reply:
It may have been installing the x86 version!

Reply:
It seems to be working now.